<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Base64CoderDecoderWidget</class>
 <widget class="QWidget" name="Base64CoderDecoderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="urlSafeCheckBox">
       <property name="text">
        <string>Url safe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Text</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openInputButton">
           <property name="toolTip">
            <string>Open</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyInputButton">
           <property name="toolTip">
            <string>Copy</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditCopy"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pasteInputButton">
           <property name="toolTip">
            <string>Paste</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditPaste"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearInputButton">
           <property name="toolTip">
            <string>Clear</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditClear"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="text"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Base64</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openCodedButton">
           <property name="toolTip">
            <string>Open</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="copyCodedButton">
           <property name="toolTip">
            <string>Copy</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditCopy"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pasteCodedButton">
           <property name="toolTip">
            <string>Paste</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditPaste"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearCodedButton">
           <property name="toolTip">
            <string>Clear</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="QIcon::ThemeIcon::EditClear"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="base64"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
